data_invariants.tex

```diff
@@ -1,5 +1,5 @@
 \begin{syntax}
-  declaration ::= "/*@" data-inv-decl "*/"
+  acsl-annot ::= data-inv-decl
```

Comment:
- #60 renames `axiom-decl` in `axiom-def`. Isn't be possible to do the same kind of renaming with `data_inv-decl` ?

- I woud prefer to do not introduce the new rule word àcsl-annot`. It was not necessary for axioms. It should be the same for `data invariant`

Reply:
Fixed in 57148886b3e15ea52e08bc6d51e35010e1ba3aaf